docs/sp/fields.md

```diff
@@ -254,6 +254,31 @@ const r = await field.field.select("Id")();
 console.log(r.Id);
 ```
 
+### Add an Image Field
+
+Use the addImageField method to create a new image field.
+
+```TypeScript
+import { spfi } from "@pnp/sp";
+import { IFieldAddResult, FieldTypes } from "@pnp/sp/fields/types";
+import "@pnp/sp/webs";
+import "@pnp/sp/lists";
+import "@pnp/sp/fields";
+
+const sp = spfi(...);
+
+// create a new image field called 'My Field' in web.
+const field: IFieldAddResult = await sp.web.fields.addImageField("My Field", { FieldTypeKind:FieldTypes.Image });
```

Comment:
The way addImageField works you shouldn't need to add the object with FieldTypeKind... that was the point of addImageField that it would do that, which ATM it's passing 34 through...

Reply:
Indeed. You are correct. This is not needed. Removed it from the Docs and Test.